ungleich_page/models.py

```diff
@@ -222,3 +222,24 @@ class UngleichHTMLOnly(CMSPlugin):
 
     def __str__(self):
         return self.name
+
+
+class UngleichFooter(CMSPlugin):
+    copyright = models.CharField(max_length=100, default='', blank=True)
```

Comment:
Probably add a help line explaining that this is name of the company alongside the copyright year